### ISSUE TITLE
Update to version 3.0.3

### ISSUE THREAD

**Issue by tGeanezini** [TrekDev/Xamarin.Android.Skobbler#14]
Hello, 
I'd like to know if you have plans on updating the project to the version 3.0.3 of the Skobbler SDK. Currently the version 3.0.2 has some lag issues when navigating using devices running Android 7+.

Thanks in advance

**Comment by jeremy-ellis-tech** [comment]
Yep I'll update it when I get the chance. Might take a little longer than ususual since I don't use Windows regularly anymore. Thanks for letting me know. I don't follow Skobbler updates as closely as I used to so getting a message when a new version is out is helpful. Thanks.

**Comment by jeremy-ellis-tech** [comment]
Also I can't seem to get the latest version. The downoad page takes me here: https://developer.skobbler.com/#takeoff

Do you have the demo app source + .apk download for 3.0.3 as it appeared on the Skobbler page?

**Comment by tGeanezini** [comment]
Hi, thanks for the quick response. It seems that the hotfix project can be downloaded at this link:
[http://share.skobbler.com/android/PublicSDK3.0.3/Hotfix/AndroidDemoPublicHotfix.zip](http://share.skobbler.com/android/PublicSDK3.0.3/Hotfix/AndroidDemoPublicHotfix.zip). It's an Android Studio project. 

I've tried changing the libraries in your project with the ones on the hotfix but I guess it doesn't work that way.

Thanks again for the help.

**Comment by jeremy-ellis-tech** [comment]
By llibraries do you mean the `.so` files? It should be as simple as swapping those out if that's all that needs doing.

**Comment by tGeanezini** [comment]
Actually, I haven't tried to swap the `.so` files, just the `.jar` and `.aar`. Let me try that and see what the result is.

**Comment by tGeanezini** [comment]
Hi again, I've tried swapping the `.so` files but the navigation kept on lagging with Android 7+, I think there's maybe more that need to be done than just swap the files.

**Comment by Jeaan2** [comment]
Hello! 
The version 3.1 seems to be a lot better. Is there a possibility to uptade to that version?

**Comment by schmitzfabio** [comment]
Hey!

It would be realy great if you could please provide the adapted Version for the 3.1 Framework. Thanks!

**Comment by schmitzfabio** [comment]
You can find the new demo projects and SDKs for Android and iOS in our dedicated repository. 

http://post.spmailtechn.com/f/a/fMMMeHiJ3YTM86crddTREQ~~/AACRFAA~/RgRcqLr2P0QgaHR0cHM6Ly9hcnRlZmFjdHMxLnNrb2JibGVyLm5ldC9XA3NwY1gEAAAAAEIKAAn4NcZaZMiGpFIRZnNjaG1pdHpAYWVpbnMuZGU~

For log in, please use the following credentials:
•	username: msk-client
•	password: Shvywkdy7r
